Vic2ToHoI4/Source/HOI4World/HoI4FocusTree.cpp

```diff
@@ -2063,6 +2063,283 @@ void HoI4FocusTree::addGPWarBranch(shared_ptr<HoI4::Country> Home,
 	nextFreeColumn += 2 * static_cast<int>(max(newAllies.size(), GCTargets.size()));
 }
 
+std::map<std::string, int> HoI4FocusTree::determineCoreHolders(std::shared_ptr<HoI4::Country> theCountry,
+	 const std::map<int, HoI4::State>& states
+){
+	std::map<std::string, int> coreHolders;
+
+	for (const auto& coreState: theCountry->getCoreStates())
+	{
+		if (const auto& owner = states.find(coreState)->second.getOwner(); owner != theCountry->getTag())
+		{
+			if (theCountry->isEligibleEnemy(owner))
+			{
+				int numProvinces = std::min(static_cast<int>(states.find(coreState)->second.getProvinces().size()), 10);
+				if (auto coreHolder = coreHolders.find(owner); coreHolder == coreHolders.end())
```

Comment:
Instead of the if/else block:
```
const auto& [existing, inserted] = coreHolders.insert(make_pair(owner, numProvinces));
if (!inserted)
{
  existing->second += numProvinces;
}
```